sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,49 @@
+COMMIT_ID="23b651d1e9c6ae1454e5d3fe315d7d4e145661c0"
+SUMMARY="A small tool which tries to guess what partitions are on a PC type"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are on a PC type, \
+MBR-partitioned hard disk in case the primary partition table was damaged."
```

Comment:
```diff
-DESCRIPTION="Gpart is a small tool which tries to guess what partitions are on a PC type, \
-MBR-partitioned hard disk in case the primary partition table was damaged."
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
```